src/system/types/cosmere.ts

```diff
@@ -282,3 +282,8 @@ export const enum TurnSpeed {
     Fast = 'fast',
     Slow = 'slow',
 }
+
+export const enum Theme {
+    Default = 'default',
+    Stormlight = 'stormlight',
```

Comment:
Can we just add a note to this that we'll have to move this out of the core system eventually.

Reply:
Only the stormlight bit needs to move out of the core system right? But yeah will do